djangocms_versioning/admin.py

```diff
@@ -108,6 +100,36 @@ def content_link(self, obj):
     content_link.short_description = _('Content')
     content_link.admin_order_field = 'content'
 
+    def grouper_form_view(self, request):
+        """Displays an intermediary page to select a grouper object
+        to show versions of.
+        """
+        context = dict(
+            self.admin_site.each_context(request),
+            opts=self.model._meta,
+            form=grouper_form_factory(self.model._source_model)(),
+        )
+        return render(request, 'djangocms_versioning/admin/grouper_form.html', context)
+
+    def changelist_view(self, request, extra_context=None):
+        if not request.GET:
+            opts = self.model._meta
+            return redirect(reverse('admin:{}_{}_grouper'.format(
+                opts.app_label,
+                opts.model_name,
+            )))
+        return super().changelist_view(request, extra_context)
+
+    def get_urls(self):
+        info = self.model._meta.app_label, self.model._meta.model_name
+        return [
+            url(
+                r'^grouper/$',
+                self.admin_site.admin_view(self.grouper_form_view),
```

Comment:
You probably should be testing for the admin permissions on this url (it's the sort of thing that's easy to miss).